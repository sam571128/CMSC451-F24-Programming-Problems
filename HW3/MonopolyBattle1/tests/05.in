100 100 1000
36 15
24 57
74 81
23 39
93 34
50 60
77 96
72 58
96 29
49 90
38 52
88 97
21 35
44 78
11 72
96 65
63 37
59 82
19 71
68 83
77 70
19 66
77 92
18 4
57 74
100 21
57 22
86 49
88 7
17 29
85 19
13 40
58 56
46 78
84 76
54 97
77 1
36 55
25 90
67 96
90 56
15 94
5 78
11 8
94 29
54 3
83 41
15 86
53 42
57 17
30 51
55 29
30 59
85 57
62 39
13 63
36 49
95 22
72 32
65 22
4 10
8 43
94 36
77 9
33 58
53 21
43 89
73 67
74 22
99 15
83 2
87 20
97 24
42 23
48 81
96 93
23 13
5 5
9 14
27 71
1 84
20 8
85 20
32 60
67 56
20 94
96 55
94 15
31 62
98 35
40 63
86 15
78 58
15 58
68 35
54 76
5 13
88 81
32 46
66 63
