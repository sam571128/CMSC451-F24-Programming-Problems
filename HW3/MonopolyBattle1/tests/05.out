178 103 103 147 103 333 112 240 156 147 483 271 150 103 103 103 526 150 103 174 191 103 132 164 103 132 132 147 153 132 103 103 103 103 147 112 103 103 163 103 198 150 103 103 123 103 111 103 132 132 103 150 262 103 104 103 256 103 367 152 150 140 119 150 103 147 103 103 147 103 132 132 150 147 103 216 202 103 483 560 195 103 150 121 297 103 171 103 381 132 144 111 121 103 115 119 147 103 112 134 